plugins/javadoc/src/main/resources/views/components/head.korte

```diff
@@ -1,19 +1,20 @@
 <head>
-    <!-- Generated by javadoc (11.0.2) on Wed Mar 25 12:07:03 CET 2020 -->
     <title>{{ title }} ({{ docName }} {{ version }})</title>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <meta name="dc.created" content="2020-03-25">
     <link rel="stylesheet" type="text/css" href="{{ pathToRoot }}stylesheet.css" title="Style">
-    <link rel="stylesheet" type="text/css" href="{{ pathToRoot }}jquery/jquery-ui.css" title="Style">
-    <script src="{{ pathToRoot }}tag-search-index.js"></script>
-    <script src="{{ pathToRoot }}jquery/jszip/dist/jszip.js"></script>
-    <script type="text/javascript" src="{{ pathToRoot }}script.js"></script>
-    <script type="text/javascript" src="{{ pathToRoot }}jquery/jszip/dist/jszip.min.js"></script>
-    <script type="text/javascript" src="{{ pathToRoot }}jquery/jszip-utils/dist/jszip-utils.min.js"></script>
-    <!--[if IE]>
-    <script type="text/javascript" src="{{ pathToRoot }}jquery/jszip-utils/dist/jszip-utils-ie.min.js"></script>
-    <![endif]-->
-    <script type="text/javascript" src="{{ pathToRoot }}jquery/jquery-3.3.1.js"></script>
-    <script type="text/javascript" src="{{ pathToRoot }}jquery/jquery-migrate-3.0.1.js"></script>
-    <script type="text/javascript" src="{{ pathToRoot }}jquery/jquery-ui.js"></script>
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.css" title="Style">
```

Comment:
Prove me wrong, but imho it is better to have dependencies downloaded from CDN and not hard-coded in repository